priiloader/hacks_hash.ini

```diff
@@ -272,54 +381,71 @@ hash=0x38000000,0x2c000000,0x40820014,0x38
 patch=0x38000001
 hash=0x9421ffd0,0x7c0802a6,0x90010034,0x38610008
 patch=0x4E800020
-[Force Disc Games to run under IOS249]
+[No System Menu Sounds AT ALL]

```

Comment:
is this a new hack?

Reply:
No, it's just the git diff being stupid.